tests/unit/src/util/Git.ts

```diff
@@ -105,7 +97,7 @@ registerSuite({
 			assert.strictEqual(error.message,
 				'Repository mismatch. Expected "url" to be "other_url".');
 		});
-		assert.doesNotThrow(async () => await git.assert('url'));
+		assert.doesNotThrow(() => git.assert('url'));
```

Comment:
Sorry, I misunderstood your previous request, and took the `return` to be *from* the anonymous function, not the test itself. Now I understand that if it returns an assertion and that doesn't pass, the test will fail. 🤦‍♂️